ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsService.scala

```diff
@@ -176,12 +191,16 @@ class ContractsService(
   def retrieveAll(
       jwt: Jwt,
       jwtPayload: JwtPayload,
+  )(implicit
+      lc: LoggingContextOf[CorrelationID with Any]
   ): SearchResult[Error \/ domain.ActiveContract[LfValue]] =
     retrieveAll(jwt, jwtPayload.parties)
 
   def retrieveAll(
       jwt: Jwt,
       parties: OneAnd[Set, domain.Party],
+  )(implicit
+      lc: LoggingContextOf[CorrelationID with Any]
```

Comment:
Isn’t that equivalent to just `CorrelationID`?

Reply:
yep you're right. I'll remove that crap, learned something again about Scala.